packages/firestore/test/integration/api/test_bundles.json

```diff
@@ -0,0 +1 @@
,\"structuredQuery\":{\"from\":[{\"collectionId\":\"coll-1\"}],\"orderBy\":[{\"field\":{\"fieldPath\":\"bar\"},\"direction\":\"DESCENDING\"},{\"field\":{\"fieldPath\":\"__name__\"},\"direction\":\"DESCENDING\"}],\"limit\":{\"value\":1}},\"limitType\":\"LAST\"}}}148{\"documentMetadata\":{\"name\":\"projects/test-emulator/databases/(default)/documents/coll-1/a\",\"readTime\":{\"seconds\":1000,\"nanos\":9999},\"exists\":true}}219{\"document\":{\"name\":\"projects/test-emulator/databases/(default)/documents/coll-1/a\",\"createTime\":{\"seconds\":1,\"nanos\":9},\"updateTime\":{\"seconds\":1,\"nanos\":9},\"fields\":{\"k\":{\"stringValue\":\"a\"},\"bar\":{\"integerValue\":1}}}}148{\"documentMetadata\":{\"name\":\"projects/test-emulator/databases/(default)/documents/coll-1/b\",\"readTime\":{\"seconds\":1000,\"nanos\":9999},\"exists\":true}}219{\"document\":{\"name\":\"projects/test-emulator/databases/(default)/documents/coll-1/b\",\"createTime\":{\"seconds\":1,\"nanos\":9},\"updateTime\":{\"seconds\":1,\"nanos\":9},\"fields\":{\"k\":{\"stringValue\":\"b\"},\"bar\":{\"integerValue\":2}}}}","fir-testing-field":"125{\"metadata\":{\"id\":\"test-bundle\",\"createTime\":{\"seconds\":1001,\"nanos\":9999},\"version\":1,\"totalDocuments\":2,\"totalBytes\":1533}}379{\"namedQuery\":{\"name\":\"limit\",\"readTime\":{\"seconds\":1000,\"nanos\":9999},\"bundledQuery\":{\"parent\":\"projects/fir-testing-field/databases/(default)/documents\",\"structuredQuery\":{\"from\":[{\"collectionId\":\"coll-1\"}],\"orderBy\":[{\"field\":{\"fieldPath\":\"bar\"},\"direction\":\"DESCENDING\"},{\"field\":{\"fieldPath\":\"__name__\"},\"direction\":\"DESCENDING\"}],\"limit\":{\"value\":1}},\"limitType\":\"FIRST\"}}}386{\"namedQuery\":{\"name\":\"limit-to-last\",\"readTime\":{\"seconds\":1000,\"nanos\":9999},\"bundledQuery\":{\"parent\":\"projects/fir-testing-field/databases/(default)/documents\",\"structuredQuery\":{\"from\":[{\"collectionId\":\"coll-1\"}],\"orderBy\":[{\"field\":{\"fieldPath\":\"bar\"},\"direction\":\"DESCENDING\"},{\"field\":{\"fieldPath\":\"__name__\"},\"direction\":\"DESCENDING\"}],\"limit\":{\"value\":1}},\"limitType\":\"LAST\"}}}152{\"documentMetadata\":{\"name\":\"projects/fir-testing-field/databases/(default)/documents/coll-1/a\",\"readTime\":{\"seconds\":1000,\"nanos\":9999},\"exists\":true}}223{\"document\":{\"name\":\"projects/fir-testing-field/databases/(default)/documents/coll-1/a\",\"createTime\":{\"seconds\":1,\"nanos\":9},\"updateTime\":{\"seconds\":1,\"nanos\":9},\"fields\":{\"k\":{\"stringValue\":\"a\"},\"bar\":{\"integerValue\":1}}}}152{\"documentMetadata\":{\"name\":\"projects/fir-testing-field/databases/(default)/documents/coll-1/b\",\"readTime\":{\"seconds\":1000,\"nanos\":9999},\"exists\":true}}223{\"document\":{\"name\":\"projects/fir-testing-field/databases/(default)/documents/coll-1/b\",\"createTime\":{\"seconds\":1,\"nanos\":9},\"updateTime\":{\"seconds\":1,\"nanos\":9},\"fields\":{\"k\":{\"stringValue\":\"b\"},\"bar\":{\"integerValue\":2}}}}"}
```

Comment:
Are the quotes meant to be escaped? Seems like they shouldn't be.

Reply:
The script is generating bundle strings, which are then put in a string-to-string map and serialize to json, to support multiple project ids. That is why the quotes are escaped here.